### ISSUE TITLE
provide a USD-price for a Niflot's contained value

### ISSUE THREAD

**Issue by elmariachi111** [elmariachi111/niflot#6]
NIflots can stream in wETH but buyers want to buy them using USDC
provide a $-value on the frontend

optionally: use an onchain (Twap) oracle, eg Uniswap to determine the current unwrapped token $ value and add a getter that yields the $-value for buyers, subtracting the discount

**Comment by joeljesu** [comment]
Yep, this is a frontend challenge. I think it could be a matter of an investor passing in a timestamp along with their bids and when they pass in that timestamp, it shows how much cashflow is left.

If their outright buying, then it's a matter that before they approve the transaction, they get a little view of what how much remaining cashflow their about to purchase.

I think regarding non-stable payment streams, there's no need to provide a estimated dollar value actually. If you have the conviction to buy an ETH cashflow, then you're making a bet that does better with numba up and worse with numba down. Getting to just leave it in ETH.

That being said, I highly doubt we'll see a WETHx salary stream anytime soon 🙃
